### ISSUE TITLE
Breakdown for blocked by client

### ISSUE THREAD

**Issue by ttrepper** [AdguardTeam/AdGuardHome#1631]
Hi all, 

as a feature request - I am using AdGuard Home v. 0.101.0 - I would love to be able to get a more detailed breakdown of the blocked DNS queries. Right now I see the statistic of the most blocked ones but I would love to do a click on such an entry and see the list of clients most asking for that. 

Thanks a lot in advance, 
Thomas

**Comment by ameshkov** [comment]
Looks like a duplicate of https://github.com/AdguardTeam/AdGuardHome/issues/1625

Please upvote that feature request if you like it
